### ISSUE TITLE
NullReferenceException in HighlightHelper (v8.9.34)

### ISSUE THREAD

**Issue by Bergam64** [sboulema/CodeNav#136]
Hello Samir,

Since I have started using v8.9.34, I see many of the following exceptions in the Output Window (part "Extensions"):

```
System.AggregateException: One or more errors occurred. ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at CodeNav.Helpers.HighlightHelper.<FindItemContainer>d__6.MoveNext() in D:\a\CodeNav\CodeNav\CodeNav.Shared\Helpers\HighlightHelper.cs:line 153
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at CodeNav.Helpers.HighlightHelper.<Highlight>d__5.MoveNext() in D:\a\CodeNav\CodeNav\CodeNav.Shared\Helpers\HighlightHelper.cs:line 106
   --- End of inner exception stack trace ---
---> (Inner Exception #0) System.NullReferenceException: Object reference not set to an instance of an object.
   at CodeNav.Helpers.HighlightHelper.<FindItemContainer>d__6.MoveNext() in D:\a\CodeNav\CodeNav\CodeNav.Shared\Helpers\HighlightHelper.cs:line 153
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
```

Consequently, auto-highlighting is broken.

I don't have a precise repro scenario for this one, but it happens often, mostly by just clicking in the C# code being edited.

Please fix this.

A little piece of advice: have you ever considered enabling C# 8 Nullable Reference Types feature in your project(s)? This would definitely help anticipating this kind of issues.

Thanks in advance,
Didier

**Comment by Bergam64** [comment]
(forgot to mention: I'm using VS 2019 v16.11.4)

**Comment by sboulema** [comment]
Thanks for reporting!

Should be fixed in the latest release (8.9.40)

I took your advice and enabled nullable, as you noticed it has caused some regressions, but lets hope it puts and end to the null ref exceptions

**Comment by Bergam64** [comment]
Hello Samir,

The issue seems to be fixed, indeed.

Best regards,
Didier
